Cargo.toml

```diff
@@ -59,10 +59,10 @@ toml               = "0.8"
 tracing            = "0.1"
 tracing-error      = "0.2"
 tracing-subscriber = "0.3"
-typed-builder      = "0.16.2"
+typed-builder      = "0.18"
 ui_test            = "0.21.2"
 visibility         = "0.1.0"
-yansi              = "1.0.0-rc.1"
+yansi              = "1.0.0-gamma"
```

Comment:
`1.0.0-rc.1` is a more recent one ([versions](https://crates.io/crates/yansi/versions))